qa/workunits/rbd/rbd_mirror_stress.sh

```diff
@@ -325,7 +325,8 @@ write_image()
     local duration=$(($RANDOM % 35 + 15))
 
     timeout ${duration}s rbd --cluster ${cluster} -p ${POOL} bench-write \
-	${image} --io-size 4096 --io-threads 8 --io-total 10G --io-pattern rand || true
+	${image} --io-size 4096 --io-threads 8 --io-total 10G --io-pattern rand \
+        --debug-rbd=20 --debug-journaler=20 2> ${TEMPDIR}/rbd-bench-write.log || true
 }
```

Comment:
Only the latest one is important since we snapshot and compare the images after each iteration.  I originally had it recording the log with an iteration string, but it didn't provide any value and it just took up space in my /tmp/